### DISCUSSION TITLE
backgroundImage automatically resized

### DISCUSSION THREAD

**Discussion by hyosang-park** [xyflow/xyflow#4704] [Q&A]
Hello, I've been trying to use ReactFlow but having some difficulty because of lack of experience.

https://codesandbox.io/p/sandbox/f9qz2g?file=%2Fsrc%2FApp.js

Here's my test code to draw a background image and I have a question.

We can check that the image is being resized automatically while we resize the browser. How can I fix the background image regardless of the size of browser.

![image](https://github.com/user-attachments/assets/69cd68f1-fa13-4e40-8bd6-e917fec19458)

![image](https://github.com/user-attachments/assets/759932dc-b174-488d-a9a1-1e21f37c0bba)

Please give me a guide to hurdle my problems. Thank you!

**Comment by bcakmakoglu** [comment]
Your wrapping div has no size and ReactFlow won't just create a size on it's own - you have to give the RF container a size one way or another.

For your sandbox it'd be enough to just do `<div style={{ height: '100vh' }}>`.

[Here's an example.](https://codesandbox.io/p/sandbox/reactflowtest-forked-djr8wh?file=%2Fsrc%2FApp.js%3A33%2C35)

**Comment by hyosang-park** [comment]
Thank you for your great and instant answer.

I followed your example but still the size of image is affected by the size of browser.

![image](https://github.com/user-attachments/assets/09d92168-5b22-42da-a00e-a6d0f8cfd83f)

![image](https://github.com/user-attachments/assets/2c9402ea-3945-4391-a57e-b0069b789e62)

Could you please give another guide to fix the image size?

I want the image size to be fixed even if I enlarge or shrink the browser.

Thank you!!!

**Reply by hyosang-park** [reply]
Dear Engineers
Please answer, please...
Is it possible below? Does ReactFlow support the ability below?
"The background image will grow or shrink only by zoom in/out action. The image size will be fixed even if the browser gets enlared or shrink."
